doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,347 @@
+ACRN Sanity Test
+################
+
+The document describes the daily sanity test for ACRN by validation team, which 
+includes a set of system level core tests of basic functionality to demonstrate 
+ACRN hypervisor’s availability. User could manually run these after setup ACRN. 
+These cases support in ``kbl-nuc-i7``.
+
+ACRN Sanity Test Cases
+######################
+
+Hypervisor
+##########
+
+Introduction
+************
+
+ST_HV_Bootloader_LaaG_VUEFI_Default
+***********************************
+
+* Description
+
+The test case focus on testing boot up a LaaG using a virtual UEFI (OVMF) firmware and the default ``launch_uos.sh``.
+
+* Prerequisites
+
+Service OS boots up normally
+
+* Test Steps
+
+1. Launch SOS
+#. Use ``OVMF.fd`` to launch clearlinux using the pre-installed default ``launch_uos.sh`` script
+
+  .. code-block:: none
+
+     $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+  .. note::
+
+     script will use uos.img in path /home/clear/uos/. if your account name is not clear
+
+     $ mkdir -p /home/clear/uos/
+
+     $ cd /home/clear/uos/
+
+     $ curl https://download.clearlinux.org/releases/xxxxx/clear/clear-xxxxx-kvm.img.xz -o uos.img.xz
+
+     $ unxz uos.img.xz
+
+
+* Expected Result
+
+LaaG boot up normally
+
+ST_HV_Memory_LaaG_Default_Partition_Check
+*****************************************
+
+* Description
+
+This test case focus on checking if the pass through memory to Guest OS and 
+check if the memory size is in a reasonable scope as launch_uos.sh defined. 
+Since the Guest OS will take a part of memory, the available memory size will 
+not be exact as what pass-through.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Boot LaaG UOS with default memory setting in launch_uos.sh
+#. Check ``cat /proc/meminfo`` and see if the available memory is close the size passed through to UOS
+
+Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS - Depends on parameter passed to acrn-dm in launch_uos.sh.
+
+ST_HV_DM_LaaG_DefaultCPU_2GMemory_Conf
+**************************************
+
+* Description
+
+This test case focus on check if the pass through memory and CPU with certain 
+value in launch_uos.sh are correct.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Create UOS with 1 core and memory=2048MB
```

Comment:
Better to indicate that we want 1 vCPU for the VM, not necessarily one core (especially since we'll be introducing CPU sharing soon).